src/main/scala/chisel3/util/Decoupled.scala

```diff
@@ -121,7 +121,6 @@ object Decoupled {
     *
     * @note unsafe (and will error) on the producer (input) side of an IrrevocableIO
     */
-  @chiselName
```

Comment:
Can you also remove the related import at the top of this file? It's the `chisel.internal.naming._` import.